### ISSUE TITLE
[xrootd: could not authorize using gsi: provider was not found]

### ISSUE THREAD

**Issue by AnthonyAportela** [JuliaHEP/UnROOT.jl#180]
When trying to access a root file, I get the following error
```
julia> f = ROOTFile("root://cmsxrootd.fnal.gov//store/group/[some_directory]/rootfile.root")
2022/08/20 18:50:53 xrootd: could not authorize:
[xrootd: could not authorize using gsi: provider was not found]
```
So usually when I see something like this, it's because I didn't initialize the voms proxy (maybe just cms specific). But I think that shouldn't be an issue, since I initialize it before running the above commands.

Any idea what this might be?

**Comment by Moelf** [comment]
For root:// protocol, we use the Go backend, so we should probably figure out what environmental variable it uses.

Any chance you can use http:// protocol and sci-token?

**Comment by Moelf** [comment]
oh yeah you need something else because:
- https://github.com/go-hep/hep/issues/250

`gsi` is not implemented

any known work around? @sbinet

**Comment by sbinet** [comment]
- implement `gsi`
- implement sci-token

probably implementing sci-token would be easier (IIRC, it has "modern" specs and uses a "modern" underlying protocol that is more in line with what Go libraries are expecting.)

**Comment by Moelf** [comment]
I don't think sites are enabling scitoken anytime soon

**Comment by Moelf** [comment]
this is fixed by #340
